orbit/cmd/orbit/shell.go

```diff
@@ -80,7 +80,7 @@ var shellCommand = &cli.Command{
 		r, _ := osquery.NewRunner(
 			osquerydPath,
 			osquery.WithShell(),
-			osquery.WithDataPath(c.String("root-dir")),
+			osquery.WithDataPath(filepath.Join(c.String("root-dir"), "shell")),
```

Comment:
Fix works. There are some differences (and duplicated code) between `orbit.go` and `shell.go` (e.g. orbit_info only registered in `orbit.go`) that we may want to fix later.